draft-ietf-core-object-security.md

```diff
@@ -920,7 +925,7 @@ Mapping and notation here is based on "Simple Form" (Section 5.4.1.1 of {{RFC807
 [HTTP response -- Proxy to HTTP Client]
 
   HTTP/1.1 200 OK
-  Object-Security: [empty]
+  Object-Security: '' (empty string)
   Body: 00 31 d1 fc f6 70 fb 0c 1d d5 ... [binary]
```

Comment:
That should be "" not '' - quoted string is defined in the HTTP headers grammar RFC